requirements.txt

```diff
@@ -25,7 +25,7 @@ pyparsing>=2.4
 python-dateutil>=2.8.1
 pytz>=2021.3
 requests>=2.20
-ruamel.yaml>=0.16,<0.17.18  # See https://github.com/great-expectations/great_expectations/issues/4152
+ruamel.yaml>=0.16,<0.18  # See https://github.com/great-expectations/great_expectations/issues/4152
```

Comment:
The deprecated method we currently use is still present up until `0.18`.
We don't need to be quite as strict with our pin.